Build failure: chiaki-ng

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

```bash
nix build "github:NixOS/nixpkgs/696af41011d1c0364dcf1843aab1999c6cccd222#chiaki-ng"
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/296133691

### Relevant log output

```console
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/bf10gcdz15ilkp3n7bq6m2z8jxk9axxr-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/share/doc/chiaki -DCMAKE_INSTALL_INFODIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/include -DCMAKE_INSTALL_SBINDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/strip -DCMAKE_RANLIB=/nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/ranlib -DCMAKE_AR=/nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/6dd5d5icf5wdv1hy9ss02fpmp8kr9kgp-chiaki-ng-1.9.6 -Wno-dev -DCHIAKI_USE_SYSTEM_CURL:STRING=true -DPROTOC_EXE=/nix/store/83m068npfi97jclcql4b0xy1220c4lh4-protobuf-30.2/bin/protoc -DProtobuf_PROTOC_EXE=/nix/store/83m068npfi97jclcql4b0xy1220c4lh4-protobuf-30.2/bin/protoc -DProtobuf_PROTOC_EXECUTABLE=/nix/store/83m068npfi97jclcql4b0xy1220c4lh4-protobuf-30.2/bin/protoc -DPROTOC_EXE=/nix/store/83m068npfi97jclcql4b0xy1220c4lh4-protobuf-30.2/bin/protoc -DProtobuf_PROTOC_EXE=/nix/store/83m068npfi97jclcql4b0xy1220c4lh4-protobuf-30.2/bin/protoc -DProtobuf_PROTOC_EXECUTABLE=/nix/store/83m068npfi97jclcql4b0xy1220c4lh4-protobuf-30.2/bin/protoc
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /nix/store/nbph466agczbny52jzk143ydcp2x14q4-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'libplacebo'
--   Found libplacebo, version 7.349.0
-- Found CURL: /nix/store/idjpxk12syd5ag1fkz7mp2rzq7picnqf-curl-8.12.1/lib/libcurl.so (found version "8.12.1") found components: HTTP HTTPS WS WSS
-- Checking for one of the modules 'speexdsp'
-- Checking for module 'speexdsp'
--   Found speexdsp, version 1.2.1
-- Speex DSP found echo cancelling and noise suppression enabled
-- Found PythonInterp: /nix/store/kjvgj2n3yn70hmjifg6y0bk9m4rf7jba-python3-3.12.10/bin/python3 (found suitable version "3.12.10", minimum required is "3")
-- Could NOT find nanopb (missing: nanopb_DIR)
-- Found Python: /nix/store/kjvgj2n3yn70hmjifg6y0bk9m4rf7jba-python3-3.12.10/bin/python3.12 (found version "3.12.10") found components: Interpreter
-- Python install dir: /nix/store/kjvgj2n3yn70hmjifg6y0bk9m4rf7jba-python3-3.12.10/lib/python3.12/site-packages
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found OpenGL: /nix/store/iszaj55ikcbnfckr6qggj7s2158glk8j-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/zi25r4vsyamsvgpvsz5yqaj3ykhv8cr3-vulkan-headers-1.4.309.0/include
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Steam shortcut enabled
-- Checking for module 'libavutil'
--   Found libavutil, version 59.39.100
-- Checking for module 'libswresample'
--   Found libswresample, version 5.3.100
-- Checking for module 'libswscale'
--   Found libswscale, version 8.3.100
-- Checking for module 'libavcodec'
--   Found libavcodec, version 61.19.101
-- Checking for module 'libavformat'
--   Found libavformat, version 61.7.100
-- Found FFMPEG: avcodec;avutil;avformat (found version "59.39.100") found components: avcodec avutil avformat
-- FFMPEG Decoder enabled
-- Could NOT find ILClient (missing: ILClient_libbcm_host_LIBRARY ILClient_libvcilcs_LIBRARY ILClient_libvchiq_arm_LIBRARY ILClient_libvcos_LIBRARY ILClient_INCLUDE_DIR ILClient_SOURCE_DIR) 
-- Pi Decoder disabled
-- Found Opus: /nix/store/1yvc2b8anrpyh73r5dwc9w25zr3yy1vl-libopus-1.5.2-dev/include
-- Checking for one of the modules 'json-c'
-- Checking for one of the modules 'miniupnpc'
-- Found OpenSSL: /nix/store/xy8x4g472i5n1bh24c5ixhbnk6qlm9vz-openssl-3.4.1/lib/libcrypto.so (found version "3.4.1")
-- Checking for one of the modules 'fftw3'
-- Found HIDAPI: /nix/store/4cif81iydy9gdmd7pi2kg26cj7zrl5qf-hidapi-0.14.0/include/hidapi  found components: hidraw
-- Steamdeck native gyro and haptics enabled
-- Found SDL2: /nix/store/3r1cplqpqfxc3j3my2ma0l311rs932qy-sdl2-compat-2.32.54-dev/lib/cmake/SDL2/SDL2Config.cmake (found version "2.32.54")
-- SDL version 2.32.54 is >= 2.0.14, disabling Setsu
-- Setsu disabled
-- Configuring done (3.6s)
-- Generating done (0.1s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025

[0m
-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j8 SHELL=/nix/store/xg75pc4yyfd5n2fimhb98ps910q5lm5n-bash-5.2p37/bin/bash
[  1%] [34m[1mGenerating takion.pb[0m
[  2%] [32mBuilding C object third-party/nanopb/CMakeFiles/protobuf-nanopb-static.dir/pb_encode.c.o[0m
[  2%] [32mBuilding C object third-party/nanopb/CMakeFiles/protobuf-nanopb-static.dir/pb_common.c.o[0m
[  2%] Built target placebo-libav-impl_autogen_timestamp_deps
[  2%] [32mBuilding C object test/CMakeFiles/munit.dir/munit/munit.c.o[0m
[  2%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf.c.o[0m
[  3%] [32mBuilding C object steamdeck_native/CMakeFiles/sdeck.dir/src/sdeck.c.o[0m
[  3%] Built target cpp-steam-tools_autogen_timestamp_deps
[  4%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_wgen.c.o[0m
[  5%] [32mBuilding C object third-party/nanopb/CMakeFiles/protobuf-nanopb-static.dir/pb_decode.c.o[0m
[  6%] [34m[1mGenerating takion.pb.c, takion.pb.h[0m
[  7%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_w4.c.o[0m
[  8%] [34m[1mAutomatic MOC for target placebo-libav-impl[0m
Traceback (most recent call last):
  File "/build/source/third-party/nanopb/generator/nanopb_generator.py", line 2761, in <module>
    main_cli()
  File "/build/source/third-party/nanopb/generator/nanopb_generator.py", line 2660, in main_cli
    results = process_file(fdesc.name, fdesc, options, other_files)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/build/source/third-party/nanopb/generator/nanopb_generator.py", line 2598, in process_file
    headerdata = ''.join(f.generate_header(includes, headerbasename, options))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/build/source/third-party/nanopb/generator/nanopb_generator.py", line 2128, in generate_header
    yield msg.fields_declaration(self.dependencies) + '\n'
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/build/source/third-party/nanopb/generator/nanopb_generator.py", line 1506, in fields_declaration
    defval = self.default_value(dependencies)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/build/source/third-party/nanopb/generator/nanopb_generator.py", line 1692, in default_value
    msg = reflection.MakeClass(desc)()
          ^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'google.protobuf.reflection' has no attribute 'MakeClass'
[  8%] Built target placebo-libav-impl_autogen
make[2]: *** [lib/protobuf/CMakeFiles/chiaki-pb.dir/build.make:74: lib/protobuf/takion.pb.c] Error 1
make[1]: *** [CMakeFiles/Makefile2:610: lib/protobuf/CMakeFiles/chiaki-pb.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  9%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_w8.c.o[0m
[  9%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_w16.c.o[0m
[ 10%] [32m[1mLinking C static library libprotobuf-nanopb.a[0m
[ 10%] Built target protobuf-nanopb-static
[ 11%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_w32.c.o[0m
[ 11%] [32m[1mLinking C static library libsdeck.a[0m
[ 12%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_w64.c.o[0m
[ 12%] Built target sdeck
[ 13%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_w128.c.o[0m
[ 13%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_rand.c.o[0m
[ 14%] [32m[1mLinking C static library libmunit.a[0m
[ 15%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_general.c.o[0m
[ 15%] Built target munit
[ 16%] [32mBuilding C object third-party/CMakeFiles/gf_complete.dir/gf-complete/src/gf_cpu.c.o[0m
[ 17%] [32m[1mLinking C static library libgf_complete.a[0m
[ 17%] Built target gf_complete
make: *** [Makefile:166: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.26, NixOS, 25.05 (Warbler), 25.05.20250504.979daf3`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.3`
 - nixpkgs: `/nix/store/708n1gvb7yizlay48z15cx51ciicvv2a-source`

### Notify maintainers

@devusb 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/games/chiaki-ng/default.nix
+++ b/pkgs/games/chiaki-ng/default.nix
@@ -23,6 +23,7 @@
   json_c,
   fftw,
   miniupnpc,
+  nanopb,
   speexdsp,
   libplacebo,
   vulkan-loader,
@@ -73,6 +74,7 @@ stdenv.mkDerivation rec {
     json_c
     fftw
     miniupnpc
+    nanopb
     libevdev
     udev
     speexdsp

